unattended_installer/install_functions/filebeat.sh

```diff
@@ -54,6 +54,8 @@ function filebeat_copyCertificates() {
             eval "tar -xf ${tar_file} -C ${filebeat_cert_path} wazuh-install-files/root-ca.pem --strip-components 1 ${debug}"
             eval "rm -rf ${filebeat_cert_path}/wazuh-install-files/ ${debug}"
         fi
+        eval "chmod 500 ${dashboard_cert_path} ${debug}"
```

Comment:
```suggestion
        eval "chmod 500 ${filebeat_cert_path} ${debug}"
```
The name of the variable is incorrect